GameData/RealismOverhaul/Engine_Configs/XLR11_Config.cfg

```diff
@@ -262,13 +339,14 @@
 	@MODULE[ModuleEngineConfigs] { @CONFIG[XLR11-RM-3] { %ratedBurnTime = #$/TESTFLIGHT[XLR11-RM-3]/ratedBurnTime$ } }
 	@MODULE[ModuleEngineConfigs] { @CONFIG[XLR11-RM-3] { %ratedContinuousBurnTime = #$/TESTFLIGHT[XLR11-RM-3]/ratedContinuousBurnTime$ } }
 }
+
 @PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[XLR11-RM-5]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
 {
 	TESTFLIGHT
 	{
 		name = XLR11-RM-5
 		ratedBurnTime = 3600		//very simple and primitive, reused extensively. Give 1 hour
-		ratedContinuousBurnTime = 360
+		ratedContinuousBurnTime = 150
```

Comment:
Extended burn time was given to allow low throttle cruise, as was performed on several X-1 missions. 150 seconds was the burn time at full throttle